src/cmake/modules/GUDHI_doxygen_target.cmake

```
@@ -44,6 +44,13 @@ if(DOXYGEN_FOUND)
     set(GUDHI_DOXYGEN_UTILS_PATH "utilities/*")
   endif()
 
+  if (DOXYGEN_VERSION VERSION_LESS 1.9.2)
```

If you are ok with it @albert-github , I would display doxygen version (for debug purpose)
```suggestion
  message("++ Doxygen version ${DOXYGEN_VERSION}")
  if (DOXYGEN_VERSION VERSION_LESS 1.9.2)
```